yt_dlp/extractor/ard.py

```diff
@@ -636,32 +636,30 @@ def _real_extract(self, url):
                     'Content-Type': 'application/json'
                 })['data']['playerPage']
 
-            content_id = str_or_none(try_get(
-                player_page, lambda x: x['tracking']['atiCustomVars']['contentId']))
+            raw_player_page = player_page
         except ExtractorError as e:
-            # try to get data from website API endpoint as public gateway failed with 503
-            if isinstance(e.cause, HTTPError) and e.cause.status == 503:
+            if not isinstance(e.cause, HTTPError) or e.cause.status != 503:
+                raise
+            else:  # try to get data from website API endpoint as public gateway failed with 503
```

Comment:
The good thing about having the `raise` in the if part above is that we can remove the `else` now, since a `raise` will abort the function at that point. This is called "early exit".
You can remove the `else` and dedent the block.